lib/webfuse/core/container_of.h

```diff
@@ -0,0 +1,16 @@
+#ifndef WF_CONTAINER_OF_H
+#define WF_CONTAINER_OF_H
+
+#ifndef __cplusplus
+#include <stddef.h>
+#else
+#include <cstddef>
+#endif
+
+#define WF_CONTAINER_OF(pointer, type, member) \
```

Comment:
Are you sure you always have statement expressions and the typeof operator?
I think this implementation is a little-bit to restrictive and does not compile anymore without gnu-extensions.
This could be an unnecessary limitation for library users.

Reply:
GNU extension stuff is removed.